.cirrus.yml

```diff
@@ -288,6 +288,9 @@ task:
     - name: Haiku x86_64
       env:
         TARGET: x86_64-unknown-haiku
+    - name: Loongnix
```

Comment:
No. Loongnix is old-world but here it's 100% sure this should be running on a new-world distro. This could be just `Linux loongarch64 lp64d`, and any adaptations for Loongnix should be done in your downstream fork specifically for Loongnix, not here.